### PR TITLE
bootstrap,s390x: Add s390x support to multiarch bootstrap build

### PR DESCRIPTION
**Pull request by brianmcarey**
There is an effort to enable support for s390x for KubeVirt[1].

This adds an s390x bootstrap image to our multiarch publish flow.

This bootstrap image does not include bazel as it will have to rely on
the s390x builder image[2] for bazel.

This image will allow us to run native s390x builds and unit tests on a
s390x cluster[3].

[1] https://github.com/kubevirt/kubevirt/pull/10490
[2] https://github.com/kubevirt/kubevirt/pull/11097
[3] https://github.com/kubevirt/project-infra/issues/3140

### CODE REVIEW

**Comment by brianmcarey** [comment]
/cc @dhiller @xpivarc 

@ibm-jitendra

**Comment by brianmcarey** [comment]
@dhiller I think this should be ok to add as s390x virtual cluster has been onboarded in prow.

File: images/bootstrap/Dockerfile [hunk 1]
```diff
@@ -60,20 +61,33 @@ RUN dnf install -y \
...
+        cd /usr/local/bin && ln -s bazelisk bazel; \
+
+        # Cache the most commonly used bazel versions inside the image
+        # and remove resulting cache directories to save a few hundred MB
+        USE_BAZEL_VERSION=4.1.0 bazel version && \
+        USE_BAZEL_VERSION=4.2.1 bazel version && \
+        USE_BAZEL_VERSION=5.3.1 bazel version && \
+        USE_BAZEL_VERSION=5.4.1 bazel version && \
+        rm -rf /root/.cache/bazel; \
+        fi
```
Comment (dhiller) [inline images/bootstrap/Dockerfile:90]: nit: could we remove one indentation step for the `fi`, so it becomes more obvious? I was looking to find it in the previous lines...

**Review by dhiller — COMMENTED** [review]
Looks great overall! I found only one nit and have one question.

**Review by dhiller — APPROVED** [review]
/approve

Thank you @brianmcarey !

File: images/bootstrap/Dockerfile [hunk 2]
```diff
@@ -60,20 +61,33 @@ RUN dnf install -y \
...
+        gcloud components install alpha beta kubectl && \
+        gcloud info | tee /workspace/gcloud-info.txt; \
+        # Cache the most commonly used bazel versions in the container
+        curl -Lo ./bazelisk https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-${ARCH} && \
+        chmod +x ./bazelisk && mv ./bazelisk /usr/local/bin/bazelisk && \
+        cd /usr/local/bin && ln -s bazelisk bazel; \
+
+        # Cache the most commonly used bazel versions inside the image
+        # and remove resulting cache directories to save a few hundred MB
+        USE_BAZEL_VERSION=4.1.0 bazel version && \
```
[the remark this answers is not in the document]
Comment (brianmcarey) [reply images/bootstrap/Dockerfile]: we updated to 5.3.1 some time ago so I think we can remove the v4 versions.
https://github.com/kubevirt/kubevirt/commit/d8e08f66039db2e1856e5415d4f141b3a4654e6f

**Pull request merged**